src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -3419,6 +3419,7 @@ public bool PerformShutdownSequence(ShutdownParams shutdownParams)
             WatchHandler.RequestSelectGeometry -= BackgroundPreviewViewModel.AddLabelForPath;
             model.ComputeModelDeserialized -= model_ComputeModelDeserialized;
             model.RequestNotification -= model_RequestNotification;
+            MainGuideManager?.CloseRealTimeInfoWindow();
```

Comment:
Thanks @jesusalvino Can you move this line to below `BackgroundPreviewViewModel.Dispose();`? The section below is for DynamoModel shutdown